src/stories/4-elements/wrapper/wrapper.stories.tsx

```diff
@@ -20,6 +20,18 @@ const meta: Meta = {
 						All-purpose component to wrap fields or add copy. This can be used to group fields, layout the
 						form or add copy in between fields.
 					</p>
+					<p>
+						When colType is set to <code>v3</code>, the columns property will be <code>ColProps</code> type
+						parameters. For a complete reference of these properties and their usage, please consult the{" "}
+						<a
+							href="https://designsystem.life.gov.sg/react/index.html?path=/docs/foundations-breakpoint-introduction--docs"
+							target="_blank"
+							rel="noopener noreferrer"
+						>
+							Breakpoint
+						</a>{" "}
+						documentation.
+					</p>
```

Comment:
this is not necessary here, the explanation should go into `COLUMNS_ARG_TYPE`

`COLUMNS_ARG_TYPE` also need to explain in v3, what are the properties / values accepted